code/modules/hydroponics/plants_flower.dm

```diff
@@ -45,9 +45,11 @@ ABSTRACT_TYPE(/datum/plant/flower)
 		var/datum/plant/P = POT.current
 		var/datum/plantgenes/DNA = POT.plantgenes	
 		var/spray_prob = max(30,(30 + DNA.endurance / 10))
+		reagents_temp = new/datum/reagents(100)
```

Comment:
100 is quite small, isn't it? Shouldn't it scale through potency/endurance?

Reply:
The cap is to hopefully prevent oversmoking as Tarmunora was worried about, though some scling would be nice, Im probably going to tie this to Yield though.

Reply:
100 isnt really all that much, a single omegaweed leaf has more chems in it than 100, and thats not even the highest end of potential chems in a plant

Reply:
Right now i have it set at 25+yield or 1, whichever is higher. Im probably gonna bump it up to 50 + yield since i have made the plant seedless, though im probably gonna keep it at that, since that puts it somewhere between a beaker and a large beaker, with the potential to surpass that